scripts/deps.sh

```diff
@@ -22,10 +22,14 @@
 ctrl_c()
 {
   exit $?
-} 
+}
 trap ctrl_c SIGINT
 
 # First load snap deps
+echo "Installing dependencies if not exists"
+go get github.com/tools/godep
+go get github.com/intelsdi-x/snap/control
```

Comment:
To be as clear as possible let's *'go get github.com/intelsdi-x/snap'* rather than a specific package within snap.